docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,124 @@
+* The target cluster must be on 5.3.x Enterprise Edition or the latest {hazelcast-cloud} release
+
+NOTE: {hazelcast-cloud} Trial and {hazelcast-cloud} Standard have a limit of 14GB of primary data. If you require more, you must use {hazelcast-cloud} Dedicated. For further information on the available {hazelcast-cloud} editions, refer to the https://docs.hazelcast.com/cloud/overview[Hazelcast {hazelcast-cloud}^] documentation.
+
+* You must specify at least one data structure name in the migration configuration file
+* All data structures specified in the migration configuration must exist in the source cluster
+* Any populated data structures that already exist on the target cluster are not migrated; however, if the existing data structure is empty, it is migrated
+* Any empty data structures are not migrated
+
+== Migrate Your Data
+
+To migrate your data, you must complete the following steps:
+
+. xref:migrate:data-migration-tool.adoc#start-the-source-cluster[Start the source cluster]
++
+The source cluster is your existing cluster, the one that you want to migrate.
++
+NOTE: Hazelcast recommends that the source cluster is put in a `PASSIVE` state before you start the migration. This is because the DMT cannot guarrantee that any data changed during migration will be migrated. For information on changing the state, see xref:maintain-cluster:cluster-member-states.adoc#changing-a-clusters-state[Changing a Cluster's State]. 
++
+If necessary, you can add data to the source cluster before continuing. For example, this can be done when testing a migration using a Development cluster. For further information on doing this, see the xref:migrate:data-migration-tool.adoc#add-data-to-cluster[Add Data to Cluster] section.
++ 
+You must also update the configuration for the source cluster and related data structures. For further information on doing this, see the xref:migrate:data-migration-tool.adoc#update-the-configuration[Update the configuration] section.
+
+. xref:migrate:data-migration-tool.adoc#check-the-target-cluster[Check the target cluster]
++
+The target cluster is the new cluster to which you want to migrate the source cluster.
+
+. xref:migrate:data-migration-tool.adoc#start-the-migration-cluster[Start the migration cluster]
++
+The cluster created by the custom Hazelcast distribution.
+
+. xref:migrate:data-migration-tool.adoc#run-the-migration[Run the Migration]
+
+. xref:migrate:data-migration-tool.adoc#verify-the-migrated-data[Verify the migrated data]
+
+NOTE: If you are using the DMT to test a migration, use a Development cluster when following the steps. 
+
+The clusters work to migrate your data as illustrated below:
+
+image::ROOT:dmt_diagram.png[DMT Clusters]
+
+==== Limited Migration Cluster License
+
+A 10-node limited license is included for use with your migration cluster. 
+
+This license is valid for 60 days following download and can be used only for data migration and trial purposes. 
```

Comment:
This is unfortunately not true (yet). It's valid 60 days from when we update it in the download package, not from the point when user downloads it.

Reply:
Brought this one up [in Slack](https://hazelcast.slack.com/archives/C053JA9B57H/p1699963536123559) again.

This is not easy to explain. The license process needs to be improved; docs are not a place to excuse or cover up the areas that are not working in a usable way....

Reply:
Updated to valid for 30 days, in line with Slack discussion